docroot/sites/brand.uiowa.edu/modules/brand_core/brand_core.module

```diff
@@ -394,7 +394,7 @@ function brand_core_node_update(Drupal\Core\Entity\EntityInterface $entity) {
         \Drupal::service('cache.render')->invalidateAll();
 
         // If available (not Local), try to clear the varnish cache for the files.
-        if (\Drupal::moduleHandler()->moduleExists('purge')) {
+        if (\Drupal::moduleHandler()->moduleExists('acquia_purge')) {
```

Comment:
FYI - I think Purge is actually the right module to check for but unfortunately we don't have a local Varnish instance running to actually purge against. Ideally, we would because it'd be useful in situations like this. So the switch to acquia_purge is just a workaround I guess for now.

Reply:
unless we move from acquia though, we will be using both so i think this is good for a while.